api/pom.xml

```diff
@@ -92,6 +91,11 @@
             <artifactId>logback-access-spring-boot-starter</artifactId>
             <version>2.7.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jaxb-annotations</artifactId>
+        </dependency>
```

Comment:
Had to add this dependency explicitly to get a successful `mvn package` - I think logstash needs this dependency.